src/main/java/seedu/mealcompanion/command/misc/RecipeAlmostCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.mealcompanion.command.misc;
+
+import seedu.mealcompanion.MealCompanionSession;
+import seedu.mealcompanion.ingredient.Ingredient;
+import seedu.mealcompanion.ingredient.IngredientList;
+import seedu.mealcompanion.recipe.Recipe;
+import seedu.mealcompanion.recipe.RecipeList;
+
+//@@author jingyaaa
+/**
+ * Represents the "recipe almost" command.
+ */
+
+public class RecipeAlmostCommand extends RecipePossibleCommand{
+
+    /**
+     * Check the number of missing ingredients to make a recipe in the ingredient list.
+     * @param recipe target recipe to check
+     * @param indredients ingredient list containing
+     * @return return number of missing ingredients to make a recipe
+     */
+
+    private int almostCanMakeRecipe(Recipe recipe, IngredientList indredients) {
```

Comment:
Maybe using a more intuitive name like missingIngredients might be better for this method